### PR TITLE
Add new BusinessExceptionType if no PoWE is found

### PR DESCRIPTION
**Pull request by 99999989**
- Added a new BusinessExceptionType NO_PROOF_OF_WORK_EFFORT_FOUND
- Use this ExceptionType when no PoWE is found

### CODE REVIEW

File: application/pom.xml [hunk 1]
```diff
@@ -46,6 +46,11 @@
             <version>${project.version}</version>
             <classifier>tests</classifier>
         </dependency>
+        <dependency>
```
Comment (pavax) [inline application/pom.xml]: @99999989 remove this?

File: domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/BusinessExceptionType.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
     WORK_EFFORT_DOES_NOT_EXIST,
     PROOF_OF_WORK_EFFORT_CLOSED,
     APPLY_DATE_NOT_IN_CONTROL_PERIOD,
+    NO_PROOF_OF_WORK_EFFORT_FOUND,
```
Comment (pavax) [inline domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/BusinessExceptionType.java]: rename to NO_MATCHING_PROOF_OF_WORK_EFFORT_FOUND

Comment (99999989) [reply domain/src/main/java/ch/admin/seco/alv/service/onlineform/domain/BusinessExceptionType.java]: Ok, makes sense

File: application/pom.xml (hunk 1 above)
Comment (99999989) [reply application/pom.xml]: Lol.. I really don't know where this was coming from. Removed.

**Pull request merged**